insights/parsers/transparent_hugepage.py

```diff
@@ -0,0 +1,67 @@
+"""
+transparent_hugepage sysfs settings
+===================================
+
+Module for parsing the sysfs settings for transparent_hugepage:
+
+ThpUseZeroPage - file ``/sys/kernel/mm/transparent_hugepage/use_zero_page``
+---------------------------------------------------------------------------
+
+Gets the contents of /sys/kernel/mm/transparent_hugepage/use_zero_page, which is either 0 or 1.
+
+Sample input::
+
+    0
+
+Examples:
+
+    >>> shared[ThpUseZeroPage].use_zero_page
+    0
+
+
+ThpEnabled - file ``/sys/kernel/mm/transparent_hugepage/enabled``
+-----------------------------------------------------------------
+
+Gets the contents of  /sys/kernel/mm/transparent_hugepage/enabled, which is something like
+`always [madvise] never` where the active value is in brackets.
+
+If no option is active (that should never happen), `get_opt()` will return None.
+
+Sample input::
+
+    always [madvise] never
+
+Examples:
+
+    >>> shared[ThpEnabled].line()
+    always [madvise] never
+    >>> shared[ThpEnabled].active_option
+    madvise
+
+"""
+
+from .. import Parser, parser
+
+
+@parser("thp_use_zero_page")
+class ThpUseZeroPage(Parser):
+    """
+    Gets the contents of /sys/kernel/mm/transparent_hugepage/use_zero_page, which is either 0 or 1.
+    """
+    def parse_content(self, content):
+        self.use_zero_page = " ".join(content).strip()
+
+
+@parser("thp_enabled")
+class ThpEnabled(Parser):
+    """
+    Gets the contents of  /sys/kernel/mm/transparent_hugepage/enabled, which is something like
+    `always [madvise] never` where the active value is in brackets.
+    If no option is active (that should never happen), get_opt() will return None.
```

Comment:
@jsvob since `get_opt` is not longer a method it should be removed.  You can add an `Attributes` section to the docstring to include the attributes in the generated documentation:
```
Attributes:
    line (str): Contents of the file input file.
    active_option (str): The active option for transparent huge pages, or `None` if not present.
```

Reply:
@bfahr Thank you pointing out my omission.